.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/web-dev/node_modules
```

Comment:
Can you please rename the root folder from `web-dev` to `src` or `lib`?
I believe those are more common names

Reply:
Sounds good, think I changed it everywhere